app/views/shared/_host_next_toc.html.erb

```diff
@@ -0,0 +1,49 @@
+<section>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-3"></div>
```

Comment:
you could remove this col-md-3, if you add the class `col-md-offset-3` to the next div